README.md

```diff
@@ -9,106 +9,236 @@
+           rest.get('/logout', (req, res, ctx) => {
+              return res(
+                ctx.json({
+                   success: true,
+                })
+              )
+           }),
+        ],
+      }
+   }
 };
+```
 
+Then, you can use other handlers in your individual story. Storybook will merge both global handlers and story handlers:
 
-// This story will include the auth handlers from preview.js
+```js
+// This story will include the auth handlers from preview.js and profile handlers
 SuccessBehavior.parameters = {
   msw: {
-      handlers: {
-          profile: rest.get('/profile', (req, res, ctx) => {
-              return res(
-                  ctx.json({
-                      firstName: 'Neil',
-                      lastName: 'Maverick',
-                  })
-              )
-          }),
-      }
+   handlers: {
+    profile: rest.get('/profile', (req, res, ctx) => {
+      return res(
+       ctx.json({
+        firstName: 'Neil',
+        lastName: 'Maverick',
+       })
+      )
+    }),
+   }
   }
 }
+```
 
+Now suppose you want to ovewrite the global handlers for auth. All you have to do is set them again in your story and these values will take precedence:
+```js
 // This story will overwrite the auth handlers from preview.js
 FailureBehavior.parameters = {
-    msw: {
-        handlers: {
-            auth: rest.get('/login', (req, res, ctx) => {
-                return res(ctx.status(403))
-            }),
-        }
-    }
+  msw: {
+   handlers: {
+    auth: rest.get('/login', (req, res, ctx) => {
+      return res(ctx.status(403))
+    }),
+   }
+  }
 }
 
+```
+
+What if you want to disable global handlers? All you have to do is set them as null and they will be ignored for your story:
+```js
 // This story will disable the auth handlers from preview.js
 NoAuthBehavior.parameters = {
-    msw: {
-        handlers: {
-            auth: null,
-            others: [
-                rest.get('/numbers', (req, res, ctx) => {
-                    return res(ctx.json([1, 2, 3]))
-                }),
-                rest.get('/strings', (req, res, ctx) => {
-                    return res(ctx.json(['a', 'b', 'c']))
-                }),
-            ],
-        }
-    }
+  msw: {
+   handlers: {
+    auth: null,
+    others: [
+      rest.get('/numbers', (req, res, ctx) => {
+       return res(ctx.json([1, 2, 3]))
+      }),
+      rest.get('/strings', (req, res, ctx) => {
+       return res(ctx.json(['a', 'b', 'c']))
+      }),
+    ],
+   }
+  }
 }
 ```
 
-[Full Documentation](https://msw-sb.vercel.app/)
+#### Configuring MSW
+
+`msw-storybook-addon` starts MSW with default configuration. If you want to configure it, you can pass options to the `initialize` function. They are the [StartOptions](https://mswjs.io/docs/api/setup-worker/start) from setupWorker.
+
+A common example is to configure the `onUnhandledRequest` behavior, as MSW logs a warning in case there are requests which were not handled.
```

Comment:
We may consider linking to the `onUnhandledRequest` page in the [docs](https://mswjs.io/docs/api/setup-worker/start#onunhandledrequest).

Reply:
good idea